packages/math-input/src/components/input/cursor-contexts.ts

```diff
@@ -10,35 +10,28 @@
  * the radical.
  */
 
-export type CursorContext = // The cursor is not in any of the other viable contexts.
```

Comment:
This was a weird file.  We exported a type union but then also a collection of string constants that represented the same values as the type union.  Now it's just a single enum!